Firebase/Auth/Source/Public/FIRPhoneAuthProvider.h

```diff
@@ -90,8 +90,7 @@ FIR_SWIFT_NAME(PhoneAuthProvider)
     @brief Starts the phone number authentication flow by sending a verifcation code to the
         specified phone number.
     @param phoneNumber The phone number to be verified.
-    @param UIDelegate A view controller object used to present the SFSafariViewController or
-        WKWebview.
+    @param UIDelegate A object used to present the SFSafariViewController or WKWebview.
```

Comment:
Switch A with An.

Also maybe we should remove WKWebView until we actually implement it.

Reply:
Both done.